app/controllers/users/request_orders_controller.rb

```diff
@@ -8,5 +8,12 @@ def show
       @request_order = current_business.request_orders.find_by(uuid: params[:uuid])
       @sub_request_orders = @request_order.children
     end
+
+    def submit
+      @request_order = current_business.request_orders.find_by(uuid: params[:uuid])
+      @request_order.update(status: 'submitted')
```

Comment:
```suggestion
      @request_order. submitted!
```
enumの場合はこちらのメソッドが使用できると思います👍

Reply:
ref: https://qiita.com/ozackiee/items/17b91e26fad58e147f2e#%E3%82%A4%E3%83%B3%E3%82%B9%E3%82%BF%E3%83%B3%E3%82%B9%E3%82%92%E6%9B%B4%E6%96%B0%E3%81%99%E3%82%8B

Reply:
ありがとうございます！牧野さんの勉強会で学んだばかりだったので、viewでは使用したのにコントローラでは早速失念してました😓記事もありがとうございます🙇‍♀️本日午前中に修正後出し直し予定です。